index.js

```diff
@@ -141,9 +140,6 @@ class DetailsDialogElement extends HTMLElement {
   static get CLOSE_SELECTOR() {
     return CLOSE_SELECTOR
   }
-  static get INPUT_SELECTOR() {
```

Comment:
Any reason why this was part of the element's API? 😕

Reply:
See #11. I should've questioned it more. 😔